### DISCUSSION TITLE
About timetable

### DISCUSSION THREAD

**Discussion by Bandana-s** [community/community#31420] [Other Feature Feedback, Questions, & Ideas]
Hi Abdul, 
What should be the format of table?
How many rows and column should be there? So that our timetable can be uploaded to the system without overlapping

**Comment by queenofcorgis** [comment]
Hi @Bandana-s 👋🏼 

Are you trying to comment on a specific post or Discussion? This Discussion is for the GitHub Community!

**Comment by Bandana-s** [comment]
Hey , I wanted to reach the developers of this project as we want to get
few idea about timetable format
